src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -13,18 +13,33 @@
                           validate_vnet, validate_vnet_required_parameters, validate_node_resource_group,
                           validate_tracing_parameters, validate_app_insights_parameters, validate_java_agent_parameters,
                           validate_instance_count)
-from ._validators_enterprise import (only_support_enterprise, validate_config_file_patterns, validate_cpu, validate_memory,
+from ._validators_enterprise import (validate_config_file_patterns, validate_cpu, validate_memory,
+                                     validate_buildpacks_binding_properties,
+                                     validate_buildpacks_binding_secrets, only_support_enterprise,
+                                     enterprise_only_and_binding_not_exist, enterprise_only_and_binding_exist,
                                      validate_git_uri, validate_acs_patterns)
 from ._utils import ApiType
 
 from .vendored_sdks.appplatform.v2020_07_01.models import RuntimeVersion, TestKeyType
+from .vendored_sdks.appplatform.v2022_05_01_preview.models \
+    import _app_platform_management_client_enums as v20220501_preview_AppPlatformEnums
+
 
 name_type = CLIArgumentType(options_list=[
     '--name', '-n'], help='The primary resource name', validator=validate_name)
 env_type = CLIArgumentType(
     validator=validate_env, help="Space-separated environment variables in 'key[=value]' format.", nargs='*')
 service_name_type = CLIArgumentType(options_list=['--service', '-s'], help='Name of Azure Spring Cloud, you can configure the default service using az configure --defaults spring-cloud=<name>.', configured_default='spring-cloud')
 app_name_type = CLIArgumentType(help='App name, you can configure the default app using az configure --defaults spring-cloud-app=<name>.', validator=validate_app_name, configured_default='spring-cloud-app')
+buildpacks_binding_properties_type = CLIArgumentType(help='Non-sensitive properties for launchProperties. '
```

Comment:
This is used only once, no need to be defined here. Otherwise the list will be tooooo loooong